topics/sequence-analysis/tutorials/annotation-with-prokka/tools.yaml

```diff
@@ -0,0 +1,9 @@
+---
+api_key: admin
+galaxy_instance: http://localhost:8080
+install_resolver_dependencies: true
+install_tool_dependencies: false
+tools:
+  - name: add_value   # name of tool in toolshed
```

Comment:
What about `prokka` and `jbrowse` there?

```
   - name: prokka
     owner: crs4
     tool_panel_section_label: "Annotation???"
   - name: jbrowse
     owner: iuc
     tool_panel_section_label: "Graph/Display Data"
```

Reply:
I have added these - I put them in the categories that they were listed in in the tool shed (prokka: Assembly;  jbrowse: Visualization)